src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
Don't use verbs for class/interface names. Consider renaming the interface to something that represents an entity or a characteristic, like `AreaCalculator`.